get-account-info-using-portal-api-keys/.env_SAMPLE

```diff
@@ -0,0 +1,5 @@
+#HEDERA PORTAL CREDENTIALS
```

Comment:
consider renaming this file to `.env.sample` (for consistency with the other snippets)